<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextToImg</class>
 <widget class="QMainWindow" name="TextToImg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文字转图片</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="childrenCollapsible">
        <bool>false</bool>
       </property>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>输入文字：</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QScrollArea" name="scrollArea">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>455</width>
          <height>416</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="showLabel">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="editMenu">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="genImgAct"/>
    <addaction name="separator"/>
    <addaction name="leftAct"/>
    <addaction name="rightAct"/>
    <addaction name="centerAct"/>
    <addaction name="justifyAct"/>
    <addaction name="separator"/>
    <addaction name="boldAct"/>
    <addaction name="italicAct"/>
    <addaction name="underAct"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="toolaBarMenu">
    <property name="title">
     <string>工具栏按钮</string>
    </property>
   </widget>
   <addaction name="editMenu"/>
   <addaction name="toolaBarMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="genImgAct"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="fontToolBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <action name="genImgAct">
   <property name="text">
    <string>&amp;生成图片</string>
   </property>
  </action>
  <action name="leftAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;左对齐</string>
   </property>
   <property name="toolTip">
    <string>左对齐</string>
   </property>
  </action>
  <action name="rightAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;右对齐</string>
   </property>
  </action>
  <action name="centerAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;居中</string>
   </property>
  </action>
  <action name="justifyAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;两端对齐</string>
   </property>
  </action>
  <action name="boldAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;加粗</string>
   </property>
  </action>
  <action name="italicAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;斜体</string>
   </property>
  </action>
  <action name="underAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;下划线</string>
   </property>
  </action>
  <action name="quitAct">
   <property name="text">
    <string>&amp;退出</string>
   </property>
  </action>
  <action name="classicAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;经典按钮</string>
   </property>
  </action>
  <action name="simpleAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;简约按钮</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
